packages/salesforcedx-vscode-apex-replay-debugger/src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,62 @@
+import vscode = require('vscode');
```

Comment:
Is there a reason we need to use `require` here instead of `import ... from .. .`?

Reply:
No, I'll update this to use import from